public/app/features/library-panels/components/LibraryPanelsView/LibraryPanelsView.tsx

```diff
@@ -23,13 +24,15 @@ export const LibraryPanelsView: React.FC<LibraryPanelViewProps> = ({
   searchString,
   showSecondaryActions,
   currentPanelId: currentPanel,
+  perPage: propsPerPage,
 }) => {
   const styles = useStyles(getPanelViewStyles);
   const [{ libraryPanels, page, perPage, numberOfPages, loadingState, currentPanelId }, dispatch] = useReducer(
     libraryPanelsViewReducer,
     {
       ...initialLibraryPanelsViewState,
       currentPanelId: currentPanel,
+      perPage: propsPerPage ?? 40,
```

Comment:
Maybe specify 40 as default value when deconstructing the properties?

Reply:
Excellent suggestion, will fix ASAP!

Reply:
Fixed